install/kubernetes/cilium/values.yaml

```diff
@@ -429,4 +431,9 @@ global:
     #
     # See https://github.com/cilium/hubble/blob/master/Documentation/metrics.md for more comprehensive
     # documentation about Hubble's metric collection.
-    metrics: []
+    metrics:
```

Comment:
If you change the type of this, you will have to adapt this line in the config map as well (at assumes that `metrics` is an array, which it now isn't anymore):
https://github.com/cilium/cilium/blob/afc48b73b65af521a7a8cb93c766dc28d379ca06/install/kubernetes/cilium/charts/config/templates/configmap.yaml#L390

Reply:
Fixed it.